content/en/docs/concepts/storage/volumes.md

```diff
@@ -7,258 +7,277 @@ reviewers:
 title: Volumes
 content_type: concept
 weight: 10
----
+--- 
```

Comment:
nit: remove extra space

Reply:
Hi @geoffcline . Thanks for creating this pull request.
I am reading through the changes.
Besides the reorganization of the volume types, there are a number of formatting and style nits.
If you want, I can mark up the nits or you can read through the style guide first and make updates.
Note that you have several headings with the same titles. This should be corrected.
I am not convinced that the new page layout improves navigation.
For example, you have two headings labelled "Core Volume Types".
I could imagine a description of the types in the "Background" section, and then
create a level H2 "Core Volume Types" section followed by the related type sections (level H3).
From my perspective, the in page lists do not add much to the page navigation and require maintenance.

The original page is organized as:
Background
Volume types
Using Subpath
Resources
"more content"
Mount Propagation
"more content"

Reply:
Thank you for your feedback.

> Note that you have several headings with the same titles. This should be corrected.

Thanks. I'll make the titles unique where appropriate, but I'm considering making some of the CSI migration headings simple bolded text, to avoid the heading issue. 

> I am not convinced that the new page layout improves navigation.
> For example, you have two headings labelled "Core Volume Types".
> I could imagine a description of the types in the "Background" section, and then
> create a level H2 "Core Volume Types" section followed by the related type sections (level H3).
> From my perspective, the in page lists do not add much to the page navigation and require maintenance.

I think you are correct that the in-text page lists don't add much. That's what the sidebar is for, and that's automatically generated too. 

I do think breaking the volume types into three categories is helpful. I think it mirrors how logically people understand volume types. 

I added a proposed ToC below, based on how I interpreted your feedback. 

```
Background
	Volume Specification Example
	🚩 Definition of the categories of volume types used below
Mount propagation
	Configuration
Specifiy subPaths on Volumes
	Using subPath with expanded environment variables
Core Volume Types
	configMap
	downwardAPI
	emptyDir
	...
Volume Plugin Interfaces
	CSI
	FlexVolume
	persistentVolumeClaim
External Volume Providers
	awsElasticBlockStore
	azureDisk
	...
What's next
```

> If you want, I can mark up the nits or you can read through the style guide first and make updates.

I will go through make updates. I'll use your commit as a guide for the types of changes to make. I do admit -- the finer points of capitalization for kubernetes terms eludes me.